stratum/hal/lib/common/yang_parse_tree_paths.cc

```diff
@@ -3205,6 +3205,104 @@ void SetUpComponentsComponentIntegratedCircuitStateNodeId(uint64 node_id,
       ->SetOnChangeHandler(on_change_functor);
 }
 
+////////////////////////////////////////////////////////////////////////////////
+// /system/logging/console/config/severity
+void SetUpSystemLoggingConsoleConfigSeverity(TreeNode* node,
+                                             YangParseTree* tree) {
+  std::string minloglevel = "UNKNOWN";
+  ::gflags::GetCommandLineOption("minloglevel", &minloglevel);
+  std::string verbosity = "UNKNOWN";
+  ::gflags::GetCommandLineOption("v", &verbosity);
+  auto poll_functor = [minloglevel, verbosity](const GnmiEvent& event,
+                                               const ::gnmi::Path& path,
+                                               GnmiSubscribeStream* stream) {
+    // This leaf represents configuration data. Return what was known when it
+    // was configured!
+    return SendResponse(
+        GetResponse(path, ConvertLogSeverityToString(minloglevel, verbosity)),
+        stream);
+  };
+
+  auto on_set_functor =
+      [node, tree](const ::gnmi::Path& path,
+                   const ::google::protobuf::Message& val,
+                   CopyOnWriteChassisConfig* config) -> ::util::Status {
+    const gnmi::TypedValue* typed_val =
+        dynamic_cast<const gnmi::TypedValue*>(&val);
+    if (typed_val == nullptr) {
+      return MAKE_ERROR(ERR_INVALID_PARAM) << "not a TypedValue message!";
+    }
+    std::string minloglevel;
+    std::string verbosity;
+    RETURN_IF_ERROR(ConvertStringToLogSeverity(typed_val->string_val(),
+                                               &minloglevel, &verbosity));
+
+    // Set the value.
+    CHECK_RETURN_IF_FALSE(
+        !::gflags::SetCommandLineOption("stderrthreshold", minloglevel.c_str())
+             .empty());
+    CHECK_RETURN_IF_FALSE(
+        !::gflags::SetCommandLineOption("minloglevel", minloglevel.c_str())
+             .empty());
+    CHECK_RETURN_IF_FALSE(
+        !::gflags::SetCommandLineOption("v", verbosity.c_str()).empty());
+
+    // Update the YANG parse tree.
+    auto poll_functor = [minloglevel, verbosity](const GnmiEvent& event,
+                                                 const ::gnmi::Path& path,
+                                                 GnmiSubscribeStream* stream) {
+      // This leaf represents configuration data. Return what was known when it
+      // was configured!
+      return SendResponse(
+          GetResponse(path, ConvertLogSeverityToString(minloglevel, verbosity)),
+          stream);
+    };
+    node->SetOnTimerHandler(poll_functor)->SetOnPollHandler(poll_functor);
+
+    // TODO(max): do we need this?
+    // Trigger change notification.
+    // tree->SendNotification(GnmiEventPtr(
+    //     new PortHealthIndicatorChangedEvent(node_id, port_id, typed_state)));
```

Comment:
I don't know who is going to listen to this. Let's wait to implement the on change stuff until someone requests it in the future.